src/tools/pbs_upgrade_job.c

```diff
@@ -154,14 +154,8 @@ typedef struct taskfix_PRE19
 	int ti_exitstat;		/* exit status */
 	union {
 		int ti_hold[16]; 	/* reserved space */
-		struct
-		{
-#ifdef _SX
-			pid_t ti_parent;
-			id_t ti_jid;
-#else
+		struct {
 			int ti_jj;	/* has to have a member */
```

Comment:
Now that there is only one element in the struct, consider removing the struct and promoting its member to the parent union.

Reply:
I actually think you can remove the structure altogether.  We had a structure with 2 elements on one side of the ifdef.  It would need at least one on the other side of the else for the code to compile.  According to my IDE, ti_jj is never used.  As a note, there are two copies of this structure (the other in job.h).